frontend/views/containers/proposals/ProposalsWidget.vue

```diff
@@ -53,6 +53,10 @@ export default ({
     ProposalItem,
     ButtonDropdownMenu
   },
+  created () {
+    // check if there are expired proposals to archive
+    sbp('gi.actions/group/checkAndArchiveExpiredProposals', { contractID: this.$store.state.currentGroupId })
+  },
```

Comment:
This will result in a GIMessage being created each time this widget is loaded. That would flood the contract chain.

Please only send any messages if and when they are necessary.